source/Mlos.Notebooks/SmartCacheOptimization.ipynb

```diff
@@ -69,7 +70,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "logger = create_logger('Optimizing Smart Cache')\n",
+    "logger = create_logger('Optimizing Smart Cache', logging_level=logging.WARN)\n",
```

Comment:
Just to be clear, this is basically the substantive change, correct?

Everything else is just updated execution output?

Reply:
yes indeed.